shared/src/test/diff/nu/BadUCS.mls

```diff
@@ -72,14 +72,8 @@ fun foo(x) = if x is M() then 0
 //│   if expression was not desugared
 
 
-:e
 fun foo0(x, y) = if x is y then 0
-//│ ╔══[ERROR] Cannot find constructor `y` in scope
-//│ ║  l.76: 	fun foo0(x, y) = if x is y then 0
-//│ ╙──      	                         ^
-//│ fun foo0: (anything, anything,) -> error
-//│ Code generation encountered an error:
-//│   if expression was not desugared
+//│ fun foo0: (anything, anything,) -> 0
```

Comment:
This is desugared into `let y = x in 0` since the expression doesn't indicate any branches.

Reply:
Ah, got it! Seems like this would warrant a warning (whose precise specification should be determined later) because of how confusing it looks. Plz add to the backlog.

Reply:
Sure. I added this to the backlog.